src/base.ts

```diff
@@ -29,6 +34,95 @@ export abstract class Base {
     this.baseUrl = "http://localhost:3000";
     this.disabledDataSettings = DisableDataSettings.None;
     this.enabled = true;
+    this.appDetails = {
+      platform_id: null,
+      app_version: null,
+      server_version: null,
+      store_id: null,
+      source_id: null,
+    };
+    this.anonId = this.generateAnonId()
+    this.userDetails = {
+      user_id: this.anonId,
+      email: null,
+      wallet: null
+    }
+  }
+
+  protected generateAnonId(): any {
+
+    let hash: any = CryptoJS.SHA256(`helika_${v4()}`);
```

Comment:
you don't need to add `helika_` here. CryptoJS.SHA256(v4()) should be fine I think